swift/Ice/greeter/Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version: 5.9
+
+import Foundation
+import PackageDescription
+
+guard let iceHome = ProcessInfo.processInfo.environment["ICE_HOME"] else {
+    fatalError("ICE_HOME environment variable not set")
+}
+
+let package = Package(
+    name: "greeter",
+    platforms: [
+        .macOS(.v14)
+    ],
+    dependencies: [.package(name: "ice", path: iceHome)],
+    targets: [
+        .executableTarget(
+            name: "GreeterClient",
+            dependencies: [.product(name: "Ice", package: "ice")],
+            exclude: ["slice-plugin.json"],
+            plugins: [.plugin(name: "CompileSlice", package: "ice")]
```

Comment:
Not directly related to the PR. Can we pass Slices sources to the `.plugin` instead of having a separate config for the plugin that we have to exclude?

Reply:
As far as I know this is not possible.